components/landing/banner/Banner.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styles from './Banner.module.scss';
+
```

Comment:
lets give this file a more descriptive name than just `Banner`.  What exactly is this component🤔, what might we call it that gives a little more info about what it is?

Reply:
I created it as the placeholder component for the mockup Banner that William has created for the landing page which should showcase some testimonials.